src/AccessibilityInsights.Extensions.GitHub/ConfigurationModelControl.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Copyright (c) Microsoft. All rights reserved.
+     Licensed under the MIT license. See LICENSE file in the project root for full license information.-->
+<src:IssueConfigurationControl
+    x:Class="AccessibilityInsights.Extensions.GitHub.ConfigurationModel"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="clr-namespace:AccessibilityInsights.Extensions.GitHub"
+    xmlns:src="clr-namespace:AccessibilityInsights.Extensions.Interfaces.IssueReporting;assembly=AccessibilityInsights.Extensions"
+     xmlns:Properties="clr-namespace:AccessibilityInsights.Extensions.GitHub.Properties"
+    mc:Ignorable="d">
+    <Canvas HorizontalAlignment="Left" Height="396" VerticalAlignment="Top" Width="360">
```

Comment:
How about using Grid? Canvas is not flexible to handle UI layout change. Grid would give you better chance to adjust UI based on the size and layout, IMO. talk to John about his experience with Canvas.